kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/internal/codec/KafkaRequestDecoder.java

```diff
@@ -144,7 +144,7 @@ static short readAcks(ByteBuf in, int startOfMessage, short apiKey, short apiVer
             if (apiVersion < 9) { // Last non-flexible version
                 transactionIdLength = in.readShort();
             }
-            else if (apiVersion == 9) { // First flexible version
+            else if (apiVersion == 9 || apiVersion == 10) { // Flexible versions
```

Comment:
Should we change this to `>= 9`? Or is it something we need to check against each new apiVersion?

Reply:
I seem to remember an earlier conversation where we agreed we wanted the prompt to check the schema.